lib/hexbin-view.js

```diff
@@ -36,7 +36,12 @@ export default class HexbinView {
           hex = '0x' + this.dec2hex(dec);
         }
         this.element.textContent = "Hex:" + hex + " Bin:" + this.formatBinary(bin) + " Dec:" + dec;
-    } else {
+      } else if (selection.length == 10 && selection.toUpperCase().substr(-10, 2) == '0B') {
+        bin = selection.substr(2, 10);
+        hex = this.bin2hex(bin);
+        dec = this.bin2dec(bin);
```

Comment:
Is there a specific reason why the dec needs to be re-set? the decimal value is already set at line 28 "dec = parseInt(selection);"